src/poprox_recommender/paths.py

```diff
@@ -2,4 +2,4 @@
 
 
 def project_root() -> Path:
-    return Path(__file__).parent.parent.parent
+    return Path(__file__).parent.parent
```

Comment:
This adjusts for models moving from `models/` to `src/models`, which changed in order to bake the models into the `whl`